MainModule/Client/UI/Default/List.lua

```diff
@@ -137,6 +137,7 @@ return function(data, env)
 			else
 				PageCounter = PageNumber
 				search.Text = ""
+				search.PlaceholderText = `Search ({#Tab} Items)`
```

Comment:
This line of code introduces a potentially experience-breaking critical bug: in a scenario when there is only one (1) item in the list, it displays "1 Items", violating an established rule of the English language and rendering the user interface practically incoherent. This has detrimental effects on our users, administrators and stakeholders alike, and may compromise the overall usability of the system. We strongly advise that you look into this issue at the earliest possible opportunity.

Reply:
blah blah blah too long didnt read

Reply:
On behalf of the entire [Astra Technologies](https://astracorp.xyz/home) team, we are currently looking into this issue.